cmd/agent/main.go

```diff
@@ -1,11 +1,10 @@
 package main
 
 import (
-	_ "sigs.k8s.io/controller-runtime/pkg/metrics"
```

Comment:
Don't modify this line. If the `sigs.k8s.io/controller-runtime/pkg/metrics` pkg hasn't initialized before `k8s.io/apiserver/pkg/server`, the metrics will be not registed to controller-runtime. See #831

Reply:
done.